src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -247,22 +259,311 @@ def get(self, request: Request, organization) -> Response:
+        help_text="A string representing the aggregate used in this alert rule. Valid aggregate functions are `count`, `count_unique`, `percentage`, `avg`, `apdex`, `failure_rate`, `p50`, `p75`, `p95`, `p99`, `p100`, and `percentile`. See [Metric Alert Rule Types](#metric-alert-rule-types) for valid configurations."
+    )
+    timeWindow = serializers.ChoiceField(
+        choices=(
+            (1, "1 minute"),
+            (5, "5 minutes"),
+            (10, "10 minutes"),
+            (15, "15 minutes"),
+            (30, "30 minutes"),
+            (60, "1 hour"),
+            (120, "2 hours"),
+            (240, "4 hours"),
+            (1440, "24 hours"),
+        ),
+        help_text="The time period to aggregate over.",
+    )
+    # projects is not required in the serializer, however, the UI requires a project is chosen
+    projects = serializers.ListField(
+        child=ProjectField(scope="project:read"),
+        help_text="The names of the projects to filter by.",
+    )
+    query = serializers.CharField(
+        help_text='An event search query to subscribe to and monitor for alerts. For example, to filter transactions so that only those with status code 400 are included, you could use `"query": "http.status_code:400"`. Use an empty string for no filter.'
+    )
+    thresholdType = serializers.ChoiceField(
+        choices=((0, "Above"), (1, "Below")),
+        help_text='The comparison operator for the critical and warning thresholds. The comparison operator for the resolved threshold is automatically set to the opposite operator. When a percentage change threshold is used, `0` is equivalent to "Higher than" and `1` is equivalent to "Lower than".',
+    )
+    triggers = serializers.ListField(
+        help_text="""
+A list of triggers, where each trigger is an object with the following fields:
+- `label`: One of `critical` or `warning`. A `critical` trigger is always required.
+- `alertThreshold`: The value that the subscription needs to reach to trigger the
+alert rule.
+- `actions`: A list of actions that take place when the threshold is met. Set as an empty list if no actions are to take place.
+```json
+triggers: [
+    {
+        "label": "critical",
+        "alertThreshold": 50,
+        "actions": [
+            {
+                "type": "slack",
+                "targetType": "specific",
+                "targetIdentifier": "#get-crit",
+                "inputChannelId": 2454362
+                "integrationId": 653532,
+            }
+        ]
+    },
+    {
+        "label": "warning",
+        "alertThreshold": 25,
+        "actions": []
+    }
+]
+```
+Metric alert rule trigger actions follow the following structure:
+- `type`: The type of trigger action. Valid values are `email`, `slack`, `msteams`, `pagerduty`, `sentry_app`, `sentry_notification`, and `opsgenie`.
+- `targetType`: The type of target the notification will be sent to. Valid values are `specific`, `user`, `team`, and `sentry_app`.
```

Comment:
It's not obvious to me what the "specific" option would do - should we add a short explanation?

Reply:
will add!